api/src/models/FedoraDataCollection.ts

```diff
@@ -172,6 +172,13 @@ class FedoraDataCollection {
         return this.fedoraDetails.sequence ?? [];
     }
 
+    get sort(): string {
```

Comment:
Should be this renamed to sortOn?

Reply:
I'm not sure if it makes a huge difference, but for the sake of consistency and specificity, I went ahead and made the change: [5369e9d](https://github.com/FalveyLibraryTechnology/VuDLPrepJS/pull/138/commits/5369e9dfdbe97e00ebe42c5887f1f6f68ee34bf0). Thanks!